nilearn/datasets/neurovault.py

```diff
@@ -0,0 +1,2589 @@
+"""
+Download statistical maps available on Neurovault (http://neurovault.org).
+"""
+# Author: Jerome Dockes
+# License: simplified BSD
+
+import os
+import logging
+import warnings
+from copy import copy, deepcopy
+import shutil
+import re
+import json
+from glob import glob
+from tempfile import mkdtemp
+from collections import Container
+try:
+    from collections import OrderedDict
+except ImportError:
+    OrderedDict = dict
+import traceback
+try:
+    from urllib.parse import urljoin, urlencode
+    from urllib.request import build_opener, Request
+    from urllib.error import URLError
+except ImportError:
```

Comment:
I believe that this is for Python 2/Python 3 support. I think that it would be good to indicate it in a comment.